workbench-core/base/src/utilities/textUtil.ts

```diff
@@ -17,5 +17,20 @@ function uuidWithLowercasePrefixRegExp(prefix: string): RegExp {
   // eslint-disable-next-line @rushstack/security/no-unsafe-regexp,security/detect-non-literal-regexp
   return new RegExp(prefix.toLowerCase() + '-' + uuidRegExpAsString);
 }
+function buildPkSk(id: string, type: string): { [key: string]: string } {
```

Comment:
I know we have unit test of all methods in here, should we add unit test for this two as well,
they may be too simple for unit test so ignore this comment if you think is not needed

Reply:
You may have missed the unit tests [here](https://github.com/aws-solutions/solution-spark-on-aws/pull/569/files#diff-3336c97ac958b7baf1cc442d0b3f9a35dee76565b0269719f9a7af0f4e4466a9R48)